### DISCUSSION TITLE
[Exercise-5]: What are anchor types and provide a screenshot of an example with result.

### DISCUSSION THREAD

**Discussion by acrlakshman** [exaslate-courses/oracle-sql-plsql-b3#6] [General]
Ref.: https://www.youtube.com/watch?v=S0XnhFOmCJ0&list=PLF6_5RI6ENIlDi3knzcQGUiFc5-WdA_1W&index=1&t=3180s

**Comment by srideviadapa0511** [comment]
Anchor data types are the data types which are assigned to the variable data types based on the data base object. unlike variable data types anchored data types are not dependent on any objects.
![ANCHOR DATA TYPES](https://github.com/exaslate-courses/oracle-sql-plsql-b3/assets/145452411/7672591f-6c69-4ddb-9100-6062312ae99e)

**Reply by acrlakshman** [reply]
@srideviadapa0511 Can you please explain what do you mean by "anchored data types are not dependent on any objects"? in your answer.

**Comment by Tharun1596** [comment]
The data type is assigned to variable based on database object is called anchor data type and it is not dependent to any object
